File: codeup.kr/1267/Main.java
import java.util.Scanner;

public class Main
{
	public static void main(String... args)
	{
		Scanner sc = new Scanner(System.in);
		int N = sc.nextInt(), result = 0;
		for(int i = 0; i < N; i++)
		{
			int tmp = sc.nextInt();
			if(tmp % 5 == 0)
			{
				result += tmp;
			}
		}
		System.out.println(result);
	}
}

File: codeup.kr/1267/main.cpp
#include <iostream>

int main(int argc, char **argv)
{
	int N, tmp, result = 0;
	std::cin >> N;
	while(int i = 0; i < N; i++)
	{
		std::cin >> tmp;
		if(tmp % 5 == 0)
		{
			result += tmp;
		}
	}
	std::cout << result << std::endl;
	return 0;
}

